src/sentry/tasks/reports.py

```diff
@@ -229,25 +230,72 @@ def build_project_series(start__stop, project):
     assert resolution == rollup, "resolution does not match requested value"
 
     clean = partial(clean_series, start, stop, rollup)
+
+    def zerofill_clean(data):
+        return clean(zerofill(data, start, stop, rollup, fill_default=0))
+
     issue_ids = project.group_set.filter(
         status=GroupStatus.RESOLVED, resolved_at__gte=start, resolved_at__lt=stop
     ).values_list("id", flat=True)
 
+    # TODO: The TSDB calls could be replaced with a SnQL call here
     tsdb_range_resolved = _query_tsdb_groups_chunked(tsdb.get_range, issue_ids, start, stop, rollup)
-    resolved_series = reduce(
+    resolved_error_series = reduce(
         merge_series,
         map(clean, tsdb_range_resolved.values()),
         clean([(timestamp, 0) for timestamp in series]),
     )
 
-    total_series = clean(
-        tsdb.get_range(tsdb.models.project, [project.id], start, stop, rollup=rollup)[project.id]
+    # Use outcomes to compute total errors and transactions
+    outcomes_query = Query(
+        dataset=Dataset.Outcomes.value,
+        match=Entity("outcomes"),
+        select=[
+            Column("time"),
+            Column("category"),
+            Function("sum", [Column("quantity")], "total"),
+        ],
+        where=[
+            Condition(Column("timestamp"), Op.GTE, start),
+            Condition(Column("timestamp"), Op.LT, stop),
+            Condition(Column("project_id"), Op.EQ, project.id),
+            Condition(Column("org_id"), Op.EQ, project.organization_id),
+            Condition(Column("outcome"), Op.IN, [Outcome.ACCEPTED]),
+            Condition(
+                Column("category"),
+                Op.IN,
+                [*DataCategory.error_categories(), DataCategory.TRANSACTION],
+            ),
+        ],
+        groupby=[Column("time"), Column("category")],
+        granularity=Granularity(rollup),
+    )
+    outcome_series = raw_snql_query(outcomes_query, referrer="reports.series")
+
+    total_error_series = [
+        (int(to_timestamp(parse_snuba_datetime(v["time"]))), v["total"])
+        for v in outcome_series["data"]
+        if v["category"] in DataCategory.error_categories()
+    ]
+    total_error_series = zerofill_clean(total_error_series)
+
+    transaction_series = [
+        (int(to_timestamp(parse_snuba_datetime(v["time"]))), v["total"])
+        for v in outcome_series["data"]
+        if v["category"] == DataCategory.TRANSACTION
+    ]
+    transaction_series = zerofill_clean(transaction_series)
+
+    error_series = merge_series(
```

Comment:
For clarity, this is the `unresolved_error_series` right?

Reply:
I think this is actually unresolved and resolved combined, so the naming is accurate